src/server/game/Spells/Spell.cpp

```diff
@@ -6392,6 +6398,303 @@ SpellCastResult Spell::CheckPetCast(Unit* target)
+            return delayForStuns;
+        // Talent - Earth's Grasp root
+        if (_spell->Id == 64695)
+            return delayForRoots;
+        break;
+
+    case SPELLFAMILY_MAGE:
+        // Deep Freeze
+        if (_spell->Id == 44572)
+            return delayForStuns;
+        // Dragon Breath
+        if (_spell->Id == 42950 || _spell->Id == 42949 || _spell->Id == 33043 || _spell->Id == 33042 || _spell->Id == 33041 || _spell->Id == 31661)
+            return delayForDisorients;
+        // Polymorph
+        if (_spell->Id == 12826 || _spell->Id == 12825 || _spell->Id == 12824 || _spell->Id == 118)
+            return delayForDisorients;
+        // Talent - Improved Counter spell silence
+        if (_spell->Id == 18469)
+            return delayForSilences;
+        // Talent - Frostbite
+        if (_spell->Id == 12494)
+            return delayForRoots;
+        // Frostnova
+        if (_spell->Id == 122 || _spell->Id == 865 || _spell->Id == 6131 || _spell->Id == 10230 || _spell->Id == 27088 || _spell->Id == 42917)
+            return delayForRoots;
+        // Pet - Freeze
+        if (_spell->Id == 33395)
+            return delayForRoots;
+            break;
+
+    case SPELLFAMILY_WARLOCK:
+        // Pet - Succubus - Seduction
+        if (_spell->Id == 6358)
+            return delayForBanishes;
+        // Banish
+        if (_spell->Id == 710 || _spell->Id == 18647)
+            return delayForBanishes;
+        // DeathCoil
+        if (_spell->Id == 6789 || _spell->Id == 17925 || _spell->Id == 17926 || _spell->Id == 27223 || _spell->Id == 47859 || _spell->Id == 47860)
+            return delayForHorrors;
+        // Pet - Felguard's intercept
+        if (_spell->Id == 47996)
+            return delayForStuns;
+        // Fear
+        if (_spell->Id == 5782 || _spell->Id == 6213 || _spell->Id == 6215)
+            return delayForFears;
+        // Howl of Terror
+        if (_spell->Id == 5484 || _spell->Id == 17928)
+            return delayForFears;
+        // Shadowfury
+        if (_spell->Id == 30283 || _spell->Id == 30413 || _spell->Id == 30414 || _spell->Id == 47846 || _spell->Id == 47847)
+            return delayForStuns;
+        // Pet - Spell Lock - Silenced
+        if (_spell->Id == 24259)
+            return delayForSilences;
+        break;
+
+    case SPELLFAMILY_DRUID:
+        // Bash
+        if (_spell->Id == 5211 || _spell->Id == 6798 || _spell->Id == 8983)
+            return delayForStuns;
+        // Cyclone
+        if (_spell->Id == 33786)
+            return delayForBanishes;
+        // Hibernate 
+        if (_spell->Id == 2637 || _spell->Id == 18657 || _spell->Id == 18658)
+            return delayForDisorients;
+        // Feral charge - Bear
+        if (_spell->Id == 45334)
+            return delayForInstantSpells;
+        // Feral charge - Cat
+        if (_spell->Id == 49376)
+            return delayForInstantSpells;
+        // Maim
+        if (_spell->Id == 22570 || 49802)
```

Comment:
``if (_spell->Id == 22570 || _spell->Id == 49802)``

Reply:
Yes, yes

Reply:
Thanks @lineagedr fixed ! 
And it passed Travis build... Strange